app/controllers/users_controller.rb

```diff
@@ -311,7 +311,7 @@ def create
       return fail_with("login.reserved_username")
     end
 
-    if user = User.where(staged: true).find_by(email: params[:email].strip.downcase)
+    if user = User.where(staged: true).find_by_email(params[:email].strip.downcase)
```

Comment:
> Usually, it is not recommended to make cosmetic changes like these as it pollutes the commit history.

I agree, but since the email column has moved from the User model to the UserEmail model, this isn't cosmetic, is it?

> Also, find_by_* have been deprecated in Rails 4 and will be removed in Rails 5.

And since we define `find_by_email` in the User model, this won't be deprecated, will it?